src/apps/AppCenter/DiscoverApps/DiscoverApps.js

```diff
@@ -1,121 +1,202 @@
-  </div>
-))
+  return (
+    <React.Fragment>
+      <Info
+        css={`
+          margin: 0 ${compactMode ? 2 * GU : 0}px;
+          margin-bottom: ${2 * GU}px;
+        `}
+      >
+        You will soon be able to install new apps into your Aragon organization
+        from here. In the meantime, you can use our{' '}
+        <Link
+          href="https://hack.aragon.org/docs/cli-dao-commands#dao-install"
+          target="_blank"
+        >
+          CLI guide
+        </Link>{' '}
+        and learn{' '}
+        <Link href="" target="_blank">
+          how to create apps yourself.
+        </Link>
+        You can also preview some of the apps being developed.
+      </Info>
+      <CardLayout columnWidthMin={columnWidthMin} rowHeight={rowHeight}>
+        {appsInDevelopment.map((app, i) => (
+          <AppCard key={i} app={app} compactMode={compactMode} />
+        ))}
+      </CardLayout>
+    </React.Fragment>
+  )
+})
 
-const AppsGrid = styled.div`
-  display: grid;
-  grid-auto-flow: row;
-  grid-gap: 25px;
-  justify-items: start;
-  grid-template-columns: 1fr;
-  ${breakpoint(
-    'medium',
-    `
-      grid-template-columns: repeat(auto-fill, 224px);
-    `
-  )};
-`
+function AppCard({ app, compactMode, onOpen, ...props }) {
+  const theme = useTheme()
+  const { link, icon, name, status, description } = app
+  const handleClick = useCallback(() => {
+    if (link) {
+      window.open(link, '_blank', 'noopener')
+    }
+  }, [link])
+
+  return (
+    <Card onClick={handleClick}>
+      <Main compactMode={compactMode}>
+        <StyledIconExternal
+          compactMode={compactMode}
+          theme={theme}
+          link={link}
+        />
+        <Icon compactMode={compactMode}>
+          <AppIcon size={9 * GU} src={icon} radius={12} />
+        </Icon>
+        <Name compactMode={compactMode}>{name}</Name>
+        <TagWrapper compactMode={compactMode} link={link}>
+          <Tag mode="indicator">{status}</Tag>
+        </TagWrapper>
+        <Description theme={theme} compactMode={compactMode}>
+          {description}
+        </Description>
+      </Main>
+    </Card>
+  )
+}
 
-const Main = styled(Card).attrs({ width: '100%', height: '288px' })`
+const Main = styled.section`
   ${unselectable};
   position: relative;
   overflow: hidden;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding-top: 25px;
+  height: 100%;
+  width: 100%;
+  white-space: initial;
+
+  ${({ compactMode }) =>
+    compactMode
+      ? `
+          display: grid;
+          grid-template-columns: auto 1fr auto;
+          grid-template-rows: auto auto auto;
+          grid-template-areas:
+            "empty topright"
+            "icon title"
+            "icon description";
+          padding: ${1.5 * GU}px ${1.5 * GU}px ${4 * GU}px ${3 * GU}px;
+        `
+      : `
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          padding-top: ${5 * GU}px;
```

Comment:
We should apply this padding to the bottom as well (or apply them both directly on the Button), otherwise the vertical alignment isn't centered and the cards start to shift around based on their content:

<img width="564" alt="Screen Shot 2019-08-26 at 7 20 15 PM" src="https://user-images.githubusercontent.com/4166642/63710187-e5c30d80-c838-11e9-9048-e87a1d1ff3f0.png">

Reply:
Verrical alignment will seem off if we allow descriptions to be of different lengths (different lengths may render in 1...N lines), we can a) push the content upwards (what I did, it allows for more lines and it is not required for all apps to have the same "length" in their descriptions) or b) Force/trim descriptions to have a min length and render a max substr of it.

Reply:
Let's stay with the current approach!